dashboard/app/lib/javascripts/dashboard/views/apps.js.jsx

```diff
@@ -16,6 +17,10 @@ var Apps = React.createClass({
 					{React.createElement(AppsList, extend({}, this.props.appsListProps, {
 						apps: this.state.apps
 					}))}
+
+					<section>
+						<SystemApplicationsLink />
```

Comment:
`this.props.showSystemApps` is [passed in from the router](https://github.com/flynn/flynn/blob/master/dashboard/app/lib/javascripts/dashboard/routers/apps.js#L93-L127). Between that and passing in a `getAppsPath` prop function similar to the `getAppPath` one we should be able to inline the whole thing here fairly easily.

I'm thinking something like:

```jsx
<RouteLink path={ this.props.getAppsPath({ system: !this.props.showSystemApps }) }>
  { this.props.showSystemApps ? 'Hide' : 'Show' } System Applications
</RouteLink>
```

(The `<RouteLink/>` component is defined [here](https://github.com/flynn/flynn/blob/master/dashboard/app/lib/javascripts/dashboard/views/route-link.js.jsx).)